sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/RoslynDataImporter.java

```diff
@@ -31,8 +31,18 @@
 
 @ScannerSide
 public class RoslynDataImporter {
-  public void importRoslynReport(Path reportPath, final SensorContext context, Map<String, List<RuleKey>> activeRoslynRulesByPartialRepoKey) {
-    final Map<String, String> repositoryKeyByRoslynRuleKey = new HashMap<>();
+  public void importRoslynReport(List<Path> reportPaths, final SensorContext context, Map<String, List<RuleKey>> activeRoslynRulesByPartialRepoKey) {
+    Map<String, String> repositoryKeyByRoslynRuleKey = getRepoKeyByRoslynRuleKey(activeRoslynRulesByPartialRepoKey);
+    SarifParserCallback callback = new SarifParserCallbackImpl(context, repositoryKeyByRoslynRuleKey);
+
+    for (Path reportPath : reportPaths) {
+      // TODO handle possible collisions
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)